### ISSUE TITLE
[Feature] <title> where can get the scores of the outputs for "outputs = model.generate(..)" or "model.stream_generate()" or "model.chat()"

### ISSUE THREAD

**Issue by wayneck** [THUDM/ChatGLM-6B#1242]
### Is your feature request related to a problem? Please describe.

I need the scores of the outputs do some choice.

### Solutions

the outputs  from  functions of 
"model.generate(..)"
"model.stream_generate()" 
 "model.chat()" all has no the items

**Comment by pjw80921** [comment]
Could you post the code of generate score? Very thankful !!!

**Comment by pjw80921** [comment]
Your meaning score is logits probs?
